sdks/python/apache_beam/utils/retry.py

```diff
@@ -46,8 +46,15 @@
 # TODO(sourabhbajaj): Remove the GCP specific error code to a submodule
```

Comment:
Would you file a JIRA issue and replace it in this line to remove the S3/GCS-specific errors from this file? We don't need to move them now, but it'd be nice to track them for later.